libs/remix-analyzer/test/analysis/staticAnalysisIntegration-test-0.5.0.ts

```diff
@@ -278,23 +278,23 @@ test('Integration test similarVariableNames module', function (t: test.Test) {
   const lengthCheck: Record<string, number> = {
     'KingOfTheEtherThrone.sol': 0,
     'assembly.sol': 0,
-    'ballot.sol': 2,
-    'ballot_reentrant.sol': 11,
+    'ballot.sol': 7,
+    'ballot_reentrant.sol': 40,
```

Comment:
so many?

Reply:
Yes. As explained in PR description, now we get waring for each similar variable name usage, previous it was for a method only even if it is using similar variable once or 100 times.